client/src/AppBundle/Controller/ClientController.php

```diff
@@ -45,7 +87,8 @@ public function showAction(Redirector $redirector)
     public function editAction(Request $request)
     {
         $from = $request->get('from');
-        $client = $this->getFirstClient();
+
+        $client = $this->clientApi->getFirstClient(null);
```

Comment:
Should this be `getFirstClient(null);`?

Reply:
Nope I don't believe so good catch 👍🏻  :)